ui/App/Theme.svelte

```diff
@@ -7,16 +7,19 @@
 -->
 <script lang="typescript">
   import { settings } from "ui/src/session";
+  import { primaryColorStore } from "ui/App/SettingsScreen.svelte";
   import { Theme, UIFont, CodeFont } from "ui/src/settings";
 
   $: {
     const theme = $settings ? $settings.appearance.theme : Theme.Dark;
+    const primaryColor = $primaryColorStore;
     const uiFont = $settings ? $settings.appearance.uiFont : UIFont.Inter;
     const codeFont = $settings
       ? $settings.appearance.codeFont
       : CodeFont.SourceCode;
     document.documentElement.setAttribute("data-theme", theme);
     document.documentElement.setAttribute("data-uifont", uiFont);
     document.documentElement.setAttribute("data-codefont", codeFont);
+    document.documentElement.setAttribute("data-fontColor", primaryColor);
```

Comment:
This allows you to remove the indirection in the assignment above.
```suggestion
    document.documentElement.setAttribute("data-fontColor", $primaryColorStore);
```